src/PerimeterxUtils.php

```diff
@@ -92,8 +92,9 @@ public static function getAllHeaders() {
      */
     public static function filterSensitiveHeaders($headers, $sensitiveHeaders) {
         $retval = [];
+        $sensitiveHeaders = array_map("strtolower", $sensitiveHeaders);
```

Comment:
cool that you lower case everything, this is a source of multiple SA escalations hhh

Reply:
Good thing we had an E2E test that checked for it!